checkov/common/vcs/base_vcs_dal.py

```diff
@@ -14,10 +14,19 @@
 
 
 class BaseVCSDAL:
+    current_branch: str  # noqa: CCE003  # a static attribute
+    current_repository: str  # noqa: CCE003  # a static attribute
+    token: str  # noqa: CCE003  # a static attribute
+    repo_owner: str  # noqa: CCE003  # a static attribute
+    graphql_api_url: str  # noqa: CCE003  # a static attribute
+    api_url: str  # noqa: CCE003  # a static attribute
+    default_branch_cache: dict[str, Any]  # noqa: CCE003  # a static attribute
+    http: urllib3.PoolManager | None  # noqa: CCE003  # a static attribute
```

Comment:
do we really want to move the attributes to the class level? or did you do it just because of type hinting? Same applies for the one in Github